apps/fe/src/libs/utils/common.ts

```
@@ -72,3 +72,34 @@ export function formatDateToHumanReadable(
+/**
+ * Sorts and stringifies an argument
+ * @param arg The argument to be sorted and stringified
+ * @returns The sorted and stringified argument as a string
+ */
+export function sortedStringify(arg: unknown): string {
+    if (typeof arg !== 'object' || arg === null || arg === undefined) {
+        return JSON.stringify(arg);
+    }
+
+    if (Array.isArray(arg)) {
+        return '[' + arg.map(sortedStringify).join(',') + ']';
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)